test/freenet/support/PrioritizedTickerTest.java

```diff
@@ -55,6 +67,71 @@ public void testSimple() throws InterruptedException {
 		}
 	}
 
+    public void testRemove() throws InterruptedException {
```

Comment:
I think the timing issues will be hard to fix however without a lot of extra and probably costly abstraction. The two existing tests in this file both rely on Thread.sleep already.